topics/ecology/tutorials/lotus2-identifying-fungi/tutorial.md

```diff
@@ -0,0 +1,462 @@
+
+We recommend signing up for a user account at one of the UseGalaxy servers (see the [first part of the Galaxy Basics tutorial]({% link topics/introduction/tutorials/galaxy-intro-101-everyone/tutorial.md %})
+
+However, if you do not have any extra time, or just want to get started with LotuS2, you can start following the steps below by following each step carefully and using the *Tips* for each *Hands-on* exercise. You can start without even creating a user account, but then you won't be able to save your histories or come back to your analysis.
+
+> <hands-on-title>Launch galaxy without a user account</hands-on-title>
+>
+> 1. Open your favorite browser (Chrome/Chromium, Safari, or Firefox, but not Internet Explorer/Edge!)
+> 2. Browse to [https://usegalaxy.eu](https://usegalaxy.eu)
+> 3. We recommend keeping this tutorial open in a separate browser window side-by-side if you have space on your desktop.
+{: .hands_on}
+
+# Understanding the files needed for running LotuS2
+
+At {SPUN}, we run LotuS2 to identify mycorrhizal fungi in a set of samples, using the following input files:
+
+1. DNA sequence files for each sample in **gzipped FASTQ** format from the Illumina MiSeq sequencer
+2. A mapping file in a **tab-separated-values** format which specifies which FASTQ files correspond to which samples
+3. A {sdm} file in **text** format which specifies how the sequence FASTQ files should be quality filtered and demultiplexed.
+
+For this tutorial we have already provided a few example files at {{ page.zenodo_link }}. You can click on this link to see which files are available (there is no need to download them).
+
+These files corresond to the three types of input files above, as shown:
+
+- DNA sequence files:
+    1. C_ITS2_S160_R1_001.fastq.gz
+    1. C_ITS2_S160_R2_001.fastq.gz
+    1. N5_ITS2_S140_R1_001.fastq.gz
+    1. N5_ITS2_S140_R2_001.fastq.gz
+    1. Pcov3_ITS2_S151_R1_001.fastq.gz
+    1. Pcov3_ITS2_S151_R2_001.fastq.gz
+- A mapping file:
+    1. Colombia_ITS2_Mapping.tsv
+- An {sdm} file
+    1. sdm_miSeq_ITS.txt
+
+For this tutorial, you do not need to download these files. Galaxy allows you to fetch data from a remote location directly into a tool or workflow without first downloading the files.
+
+In the next section we will get the data in to Galaxy, and after that we will look at the files to see what they look like.
+
+# Get Data
+
+This section describes three options that will allow you to download and use {SPUN}'s example data files. Select one option and follow the instructions.
+
+> <hands-on-title>Option 1: Data upload - Import history</hands-on-title>
+>
+> 1. Import history from: [tutorial input history](https://usegalaxy.eu/u/sujai_spun_earth/h/identifying-mf-from-its2-sequencing-using-lotus2---tutorial-input)
```

Comment:
we try to stay away from things like this that are only useful on one specific Galaxy server. 

Furthermore, all tutorial data from zenodo is made available on Galaxy EU as a shared data library, so you can also point people to that; we have an FAQ/snippet for it that you could put undernead the importing via link snippet `{% snippet faqs/galaxy/datasets_import_from_data_library.md %"`

Reply:
Hi @shiltemann and @paulzierep  - I agree that it is better to be generic!

However, because LotuS2 only works on usegalaxy.eu for now, I'd like to keep this tutorial simple for now (and edit it later to make it generic for other usegalaxy servers)

(like @paulzierep - i also tried usegalaxy.org where the tool is not present, and usegalaxy.org.au where it didn't work with the same inputs)